src/coercions/string.rs

```diff
@@ -25,3 +26,11 @@ impl ToRust<String> for CheckedValue<String> {
         unsafe { std::str::from_utf8_unchecked(slice) }.to_string()
     }
 }
+
+impl ToRuby for String {
+    fn to_ruby(self) -> VALUE {
+        let len = self.len();
+        let cstr = CString::new(self).unwrap();
```

Comment:
No need to `forget`, as that'll be a memory leak if `rb_utf8_str_new` is making its own copy of what's passed to it (as @chancancode seems to suggest it does).